### DISCUSSION TITLE
Test results overwrite

### DISCUSSION THREAD

**Discussion by marioaag** [avocado-framework/avocado#4647] [Q&A] [answered]
Is it expected to get the test results overwritten if multiple suits with the same name and the same test name are executed over the same job?

**Comment by beraldoleal** [comment]
Hi @marioaag we do have a few open issues with regards multiple jobs sending notifications to our status server, that is being fixed.

If you could share how you are defining your tests we could investigate and point any possible workaround at this stage.

**Reply by marioaag** [reply]
I'm generating sequence of parallel and sequential subsequence executions using multiple test suites, and in some cases the same test is executed in the subsequence at the begging and in the subsequence at end.
If this is expected I will proceed to add a UUI or time stamp to the suite names to avoid this issue.

Thanks!

**Comment by willianrampazzo** [comment, marked as answer]
If you are setting the same suite name with the same tests, I would say yes. If you need the logs from individual tests on different suites, try using different suite names.

**Reply by marioaag** [reply]
Ok, Thanks!
